service/src/main/java/eu/execom/labs/pomodorospringreact/service/TeamServiceImpl.java

```
@@ -121,6 +124,17 @@ public TeamDto findByTeamName(@NotNull String name) {
         return teamDtos;
     }
 
+    @Transactional(readOnly = true)
+    @Override
+    public void sendInvitation(long teamId, String email) {
+        User user = userDao.findByEmail(email);
+        if (user == null) {
+            return;
```

how do we handle this return on caller side?

---

this means that if a user that is invited to a team not a registered user nothing will happen, we could maybe throw a exception that user is not registered